### PR TITLE
14.0.27

### PR DESCRIPTION
**Pull request by willmiraglia**

### CODE REVIEW

File: api_server/modules/web/web_logout_request.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ class WebLogOutRequest extends APIRequest {
 
 		const returnTo = encodeURIComponent(this.api.config.apiServer.publicApiUrl + '/web/finish');
 		// TODO: get this from configuration
-		const loginPath = 'https://staging-login.newrelic.com/idp/azureb2c-cs/redirect?return_to=' + returnTo;
+		const loginPath = 'https://staging-login.newrelic.com/idp/azureb2c/redirect?return_to=' + returnTo;
```
Comment (bcanzanella) [inline api_server/modules/web/web_logout_request.js:17]: @cstryker @eric-j this need a url?

File: api_server/modules/web/web_login_request.js [hunk 2]
```diff
@@ -69,7 +69,7 @@ class WebLoginRequest extends APIRequest {
 		}
 
 		const returnTo = this.api.config.apiServer.publicApiUrl + finishUrl;
-		let nrLoginUrl = `https://staging-login.newrelic.com/idp/azureb2c-cs/redirect?return_to=${encodeURIComponent(returnTo)}`;
+		let nrLoginUrl = `https://staging-login.newrelic.com/idp/azureb2c/redirect?return_to=${encodeURIComponent(returnTo)}`;
```
Comment (bcanzanella) [inline api_server/modules/web/web_login_request.js:72]: @cstryker @eric-j this need a url?

File: api_server/modules/web/web_logout_request.js (hunk 1 above)
Comment (eric-j) [reply api_server/modules/web/web_logout_request.js:17]: I'll defer to Colin but I believe we need to do what the TODO says (get base url from config).

Comment (cstryker) [reply api_server/modules/web/web_logout_request.js:17]: Yeah, though I believe we don't actually use that request anymore, it should just be eliminated

File: api_server/modules/web/web_login_request.js (hunk 2 above)
Comment (cstryker) [reply api_server/modules/web/web_login_request.js:72]: This one I'm not so sure about, I'll have to do a bit more investigation

**Review by bcanzanella — APPROVED** [review]

**Pull request merged**